pkg/storage/iterators.go

```diff
@@ -58,11 +57,11 @@ func NewUniqueObjectIterator(iter1, iter2 ObjectIterator) ObjectIterator {
 
 var _ ObjectIterator = (*uniqueObjectIterator)(nil)
 
-// Next returns the next most unique object from the two underlying iterators.
-// If the context is cancelled or times out, it should return ErrIteratorDone
-func (u *uniqueObjectIterator) Next(ctx context.Context) (*openfgapb.Object, error) {
+// Next returns the next unique object from the two underlying iterators. If
+// the context is cancelled or times out, it should return ErrIteratorDone
```

Comment:
The bit about Context here is no longer needed in the godoc.

Reply:
Good catch. Fixed. Thanks!